package.json

```diff
@@ -1080,6 +1090,16 @@
                                 "pattern": "^(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])$",
                                 "type": "string"
                             },
+                            "portRangeStart": {
+                                "default": null,
+                                "description": "Start range for GDB port scan. Optional.",
+                                "type": "string"
```

Comment:
You can use `integer` here and get proper validation for free and you don't have to convert it to int.

Reply:
Adding to what @go2sh said, you can also specify a proper range. A better spec. is to ask for a number greater than 0 (0 being special) up to the max and you get auto-validation for free. Maybe we back off on the max a bit., perhaps by 2000 to match what we have.
                           
```
"portRangeStart": {
    "type": "integer",
    "default": 50000,
    "minimum": 1,
    "maximum": 65535
}
```